esx_service/utils/vmdk_utils.py

```diff
@@ -305,9 +307,21 @@ def log_volume_lsof(vol_name):
                 cartel, name, ftype, fd, desc)
             logging.info("Volume open descriptor: %s", msg)
 
+def get_vmdk_capacity(path):
```

Comment:
we already have get_vmdk_size_info in vmdkops_admin.py. You my want to move it to vmdk_utils and reuse from both places.